apps/src/templates/studioHomepages/NpsSurveyBlock.jsx

```diff
@@ -60,7 +60,7 @@ export default class NpsSurveyBlock extends React.Component {
   }
 
   onComplete(data) {
-    trackEvent('survey', 'nps2020', parseInt(data.nps_value));
+    trackEvent('survey', 'nps', parseInt(data.nps_value));
```

Comment:
We don't want to explicit track these as 2021?

Reply:
This value can be used to query the # of teachers who submit the NPS survey on Google Analytics.  I thought it would simpler if this value wasn't year-specific.  This way, I don't have to change this value every time we launch the survey and I can write a generic GA query that will work every time we run it.